src/policy.c

```diff
@@ -831,6 +831,33 @@ move_installed_to_front(Pool *pool, Queue *plist)
     }
 }
 
+
+/*
+ * prune_to_considred
```

Comment:
Oops, typo here, too. Please fix. :)

Reply:
@Conan-Kudo Thanks again, typo was fixed. You know I was so exited that it solves our critical issue that I overlooked problems in description and def name.